Source/WebCore/platform/graphics/cairo/CairoUtilities.cpp

```diff
@@ -49,13 +48,20 @@
 
 namespace WebCore {
 
-#if USE(CAIRO) && !PLATFORM(GTK)
+static auto s_defaultCairoFontOptions = CairoUniquePtr<cairo_font_options_t>();
+
 const cairo_font_options_t* getDefaultCairoFontOptions()
 {
-    static NeverDestroyed<cairo_font_options_t*> options = cairo_font_options_create();
-    return options;
+    if (!s_defaultCairoFontOptions)
+        s_defaultCairoFontOptions.reset(cairo_font_options_create());
```

Comment:
hmm, I think this could be called from multiple threads with threaded rendering or offscreen canvas enabled. Setting should always happen from the main thread, so maybe can call the set for wpe from the web process initialization like gtk does to ensure the value is always set before get is called.